service/awsconfig/v1/resource_set.go

```diff
@@ -0,0 +1,142 @@
+package v1
+
+import (
+	"github.com/giantswarm/certs/legacy"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/resource/metricsresource"
+	"github.com/giantswarm/randomkeytpr"
+	"k8s.io/client-go/kubernetes"
+
+	awsclient "github.com/giantswarm/aws-operator/client/aws"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v2/key"
+	legacyv2 "github.com/giantswarm/aws-operator/service/awsconfig/v2/resource/legacy"
+	// TODO v2 cloudconfig is used in legacy resource, v3 cloudconfig is used in v2 resources and v4 cloudconfig in v3 resources.
+	"github.com/giantswarm/aws-operator/service/awsconfig/v2/cloudconfig"
```

Comment:
Imports are uglier, but this is legacy. Goes away soon, noone is going to copy that.